testing/e2e/testkit.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+function create_git_cached_rule_for_small_repo() {
+  local commit_hash=$1
+  local cache_dir=${2:-"default-dir"}
+
+  # Trim consecutive // characters since Bazel would fail if such exists on a target path
+  local trimmed_path=$(echo ${WORKSPACE_DIR} | tr -s '/')
```

Comment:
who's populating WORKSPACE_DIR? is it a github actions thing? beware of coupling with the build server solution...

Reply:
The testing framework exports the `WORKSPACE_DIR` as appears [in here](https://github.com/wix-playground/wix_build_tools/blob/aaf49d790ae0efc5c7a2f1c2e1442e6aa0a01b09/testing/e2e/bazel.sh#L27), it relies on the temporary folder created for the testing suite.